receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -500,6 +500,13 @@ metrics:
     gauge:
       value_type: int
     attributes: []
+  k8s.pod.volume.usage:
+    enabled: false
+    description: "The number of used bytes in the pod volume."
+    unit: By
+    gauge:
```

Comment:
Based on https://github.com/open-telemetry/semantic-conventions/blob/v1.37.0/docs/system/k8s-metrics.md#metric-k8spodvolumeusage, shouldn't this be `updowncounter`?